### PR TITLE
minor/feature/UBI-494-StatsCharacters

### PR DESCRIPTION
**Pull request by MaximeStOnge**
#### What changes does this PR contain?
- Characters stats 
- Call for characters alive

#### Which JIRA ticket ID's are associated with this PR?
UBI-494

### CODE REVIEW

File: pkg/database/db_interface.go [hunk 1]
```diff
@@ -11,6 +11,7 @@ type CharacterDB interface {
 	GetCharacters(ctx context.Context) data.Characters
 	GetCharacterByID(ctx context.Context, id string) (*data.Character, error)
 	GetCharactersByUserID(ctx context.Context, userID string) (data.Characters, error)
+	GetCharactersAliveByUserID(ctx context.Context, userID string) (data.Characters, error)
```
Comment (ArcticWalrus) [inline pkg/database/db_interface.go:14]: GetAliveCharactersByUserID?

File: pkg/database/mockDB.go [hunk 2]
```diff
@@ -55,6 +55,16 @@ func (mp *MockCharacters) GetCharactersByUserID(ctx context.Context, userID stri
 	return charactersList, nil
 }
 
+func (mp *MockCharacters) GetCharactersAliveByUserID(ctx context.Context, userID string) (data.Characters, error) {
```
Comment (ArcticWalrus) [inline pkg/database/mockDB.go:58]: GetAliveCharactersByUserID?

File: pkg/handlers/get.go [hunk 3]
```diff
@@ -77,3 +77,29 @@ func (characterHandler *CharactersHandler) GetCharactersByUserID(responseWriter
 		return
 	}
 }
+
+// GET /characters/alive/user/{user_id}
+// Returns an array of characters alive for a user from the database
+func (characterHandler *CharactersHandler) GetCharactersAliveByUserID(responseWriter http.ResponseWriter, request *http.Request) {
```
Comment (ArcticWalrus) [inline pkg/handlers/get.go:83]: GetAliveCharactersByUserID?

**Review by ArcticWalrus — CHANGES_REQUESTED** [review]
Also missing at least a couple of unit tests for when the user has one, several or no alive characters

**Review by JeremiSavard — APPROVED** [review]

**Comment by ArcticWalrus** [comment]
> 
> 
> Also missing at least a couple of unit tests for when the user has one, several or no alive characters

Il ne manque que quelques tests dans handlers_test.go en utilisant la mock DB

File: pkg/handlers/get.go [hunk 4]
```diff
@@ -77,3 +77,29 @@ func (characterHandler *CharactersHandler) GetCharactersByUserID(responseWriter
 		return
 	}
 }
+
+// GET /characters/alive/user/{user_id}
+// Returns an array of characters alive for a user from the database
+func (characterHandler *CharactersHandler) GetAliveCharactersByUserID(responseWriter http.ResponseWriter, request *http.Request) {
```
Comment (ArcticWalrus) [inline pkg/handlers/get.go:83]: Missing test updates and new tests for handlers

**Review by ArcticWalrus — CHANGES_REQUESTED** [review]

**Review by ArcticWalrus — APPROVED** [review]

**Pull request merged**